.gitignore

```diff
@@ -0,0 +1 @@
+server/node_modules/
```

Comment:
We may want a `.gitignore` in the root with root related ignores, then a `.gitignore` in both the `server` and in the `client` just to keep them separate & incase we ever moved to separate repositories

Reply:
Great idea. Wow I love it when the review process yields good ideas! Collaborating is fun!